iterations_runners/iterations_runner.lua

```diff
@@ -164,7 +162,8 @@ for BM_i = 1, BM_iters, 1 do
     end
 end
 
-krun_done()
+-- Note we want a reference to libkruntime here so it's always kept alive to this point.
```

Comment:
I'd recommend a small tweak to this comment to make it more clear. How about:

> In LuaJIT, FFI functions are not considered by the garbage collector. Calling an FFI function which has been cached on the stack (as we've done above) might fail because the parent FFI object may have been GC'd . By explicitly accessing `krun_done` via `libkruntime` here we guarantee `libkruntime` to live until this point.

Reply:
Is this true of regular Lua too btw? If so, revise as required.

Reply:
Normal Lua doesn't have cdata types

Reply:
@vext01 what about this for the first sentence
> In LuaJIT, FFI functions are cdata values that are unable to reference any other object owned by the garbage collector.

Reply:
Works for me.